pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/ResourceGroup.java

```diff
@@ -20,7 +20,8 @@
 
 import lombok.Data;
 
-@Data public class ResourceGroup {
+@Data
+public class ResourceGroup {
```

Comment:
Don't need to be changed here.

Reply:
It's more readable and same with other code's style.

Reply:
Sorry, I made a mistake.